README.md

```diff
@@ -32,6 +32,14 @@ pip install .
 
 Add ``-e`` to install in [develop mode](http://pip.readthedocs.org/en/latest/reference/pip_install.html#editable-installs).
 
+``pvlib-python`` comes with Python bindings for NREL's solar position C code,
+however you must manually download the C code directly from [NREL](http://www.nrel.gov/midc/spa/).
+To use the NREL SPA algorithm,
+
+1. Download the SPA files from [NREL](http://www.nrel.gov/midc/spa/)
+2. Copy the SPA files into ``pvlib/spa_c_files``
+3. From this ``pvlib-python`` directory, run ``pip install .``
```

Comment:
Maybe it is a good idea to move the spa to a separate package, if this is easier. I think you already discussed this with @Calama-Consulting. 

This could also have the advantage, that users, that are not interessted in PV, but want to calculate solar position can install only this package.